### ISSUE TITLE
Gradle: Possible to add a folder twice

### ISSUE THREAD

**Issue by mueller-ma** [dependabot/feedback#414]
Looking at the dashboard, I saw that /mobile (https://github.com/openhab/openhab-android) isn't added:
![grafik](https://user-images.githubusercontent.com/22525368/55253861-bd69bf80-5256-11e9-90d0-81eb49ac0189.png)
So I added Java/Gradle with the directory /mobile, but then I realized, that /mobile is covered by the already existing entry:
![grafik](https://user-images.githubusercontent.com/22525368/55253930-f144e500-5256-11e9-8d52-aa03428cd005.png)

This seems to be a bug for me (adding a directory twice).

After adding the directory Dependabot did it's job and opened a PR for each not-up-to-date lib. Most of them are not up-to-date with intent, except https://github.com/openhab/openhab-android/pull/1272. I haven't seen any closed PR or any other reason why Dependabot ignores this dependency before.
https://github.com/openhab/openhab-android/pulls?utf8=%E2%9C%93&q=auto-value+from+1.6.2+to+1.6.3

**Comment by feelepxyz** [comment]
@mueller-ma sorry for the slow response on this! `/mobile` is included automatically because it's referenced from https://github.com/openhab/openhab-android/blob/master/settings.gradle

What exactly are you expecting with `auto-value` updates?

**Comment by mueller-ma** [comment]
> /mobile is included automatically because it's referenced from https://github.com/openhab/openhab-android/blob/master/settings.gradle

Yes, but in this case it shouldn't be possible to explicitly add it.

> What exactly are you expecting with auto-value updates?

Dependabot did bump `implementation 'com.google.auto.value:auto-value-annotations:1.6.3'` (#1133) before, but not `annotationProcessor 'com.google.auto.value:auto-value:1.6.3'` (#1272). It was bumped after I added `/mobile` as an additional directory.

**Comment by feelepxyz** [comment]
Ahh ok seems there's something funky going on then. Should be updating mobile if that's monitored. Will take a look.

**Comment by mueller-ma** [comment]
Poke
